frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/StrategyExecution.tsx

```diff
@@ -28,11 +27,20 @@ const NoItems: VFC = () => (
     </Box>
 );
 
+const StyledValueContainer = styled(Box)(({ theme }) => ({
+    padding: theme.spacing(2, 3),
+    border: `1px solid ${theme.palette.dividerAlternative}`,
+    borderRadius: theme.shape.borderRadiusMedium,
+}));
+
+const StyledValueSeparator = styled('span')(({ theme }) => ({
+    color: theme.palette.grey[700],
```

Comment:
We should avoid using theme.palette.grey[700] directly and use identifiers that has semantic names that makes sense. This approach leaks details about specific colors out to the components, which complicates things when we want to implement other themes. For example, in a dark theme this component is likely not going to use the grey color, but to change it we'd need to change the grey color scheme to the new color, rendering the label meaningless.